protobuf/grpc_service.proto

```diff
@@ -202,6 +202,15 @@ service GRPCInferenceService
       returns (CudaSharedMemoryUnregisterResponse)
   {
   }
+
+  //@@  .. cpp:var:: rpc Trace(TraceRequest)
+  //@@                   returns (TraceResponse)
+  //@@
+  //@@     Update and get the trace setting of the Triton server.
+  //@@
+  rpc Trace(TraceRequest) returns (TraceResponse)
```

Comment:
This should be renamed to TraceSettings since it does not get the trace but the settings for the trace

Reply:
That's a good point, in the future if we want to add other trace API we will be in trouble so we should be more specific here. TraceSettings is a good choice.